recipes/pcapplusplus/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class PcapplusplusConan(ConanFile):
+    name = "pcapplusplus"
+    version = "21.05"
+    license = "Unlicense"
+    description = "PcapPlusPlus is a multiplatform C++ library for capturing, parsing and crafting of network packets"
+    topics = ("conan", "pcapplusplus", "pcap", "network", "security", "packet")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/seladb/PcapPlusPlus"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "immediate_mode": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "immediate_mode": False,
+    }
+
+    exports_sources = "patches/*"
+    generators = "make"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Windows":
+            # FIXME: missing winpcap recipe (https://github.com/bincrafters/community/pull/1395)
+            raise ConanInvalidConfiguration("winpcap is not available on cci (yet)")
+        else:
+            self.requires("libpcap/1.9.1")
+
+    def validate(self):
+        if self.settings.os not in ("FreeBSD", "Linux", "Macos"):
```

Comment:
Does your lib compile/work on iOS/ipados/...?

Reply:
At the moment no, iOS and iPadOS are not supported